requirements/base_frozen.in

```diff
@@ -0,0 +1,19 @@
+google-cloud-storage>=1.30,<2
+# Maintainers, please do not require google-api-core>=2.x.x
+# Until this issue is closed
+# https://github.com/googleapis/google-cloud-python/issues/10566
+MarkupSafe<2.1.0 # version 2.1.0 is broken
```

Comment:
there is a version 2.1.1 of MarkupSafe available. Does it fixe the issue?

Reply:
It's still broken. I have checked that.